src/app/modules/players/players.controller.spec.js

```diff
@@ -1,11 +1,11 @@
 'use strict';
 
 describe('Players controller', function() {
-  var createCtrl;
+//  var createCtrl;
```

Comment:
Perche hai commentato il codice in questo modo? Immagino che non funzionava piu.
Qualche idea su come fixarlo piuttosto che commentarlo?